spec/ckb/types/transaction_spec.rb

```diff
@@ -132,7 +121,7 @@
                             hash_type: "data" },
                     type: nil }],
         outputs_data: %w[0x],
-        witnesses: %w(0x4107bd23eedb9f2a2a749108f6bb9720d745d50f044cc4814bafe189a01fe6fb 0x)}
+        witnesses: [CKB::Types::Witness.new(lock: "0x4107bd23eedb9f2a2a749108f6bb9720d745d50f044cc4814bafe189a01fe6fb"), "0x"]}
```

Comment:
The `lock` won't affect the signing process. Can we update the test to have 'inputType' and 'outputType' data instead?

Reply:
updated: https://github.com/nervosnetwork/ckb-sdk-ruby/pull/206/commits/9996303d9d947ba5cf049b0c878d3d1a21d90727